protocols/code-review/README.md

```diff
@@ -0,0 +1,41 @@
+# Code Review Practice
+This protocol states how we do code review in Futureworkz.
+
+## Code Review Buddy System
+Coder      | Coder
+-----------|---------
+Chacharoti | Yunchang
+Billy      | Brian
+Oliver     | Smith
+James      | Robin
+Wayne      | Theo
+
+The assigned coder buddy will be the first coder to approach for code review eg. Billy will find Brian first for code review.
+
+In the event where the coder buddy is unavailable, the coder will find either of the two coders in the next row eg. Billy will find Oliver or Smith if Brian is unavailable. Wayne will find Chacharoti or Yunchang if Theo is unavailable.
+
+In the event where the secondary reviewers are not available, the coder can find anyone to review his code.
+
+## What to Review
+We code review everything including bug fixes, HTML/CSS/JS and of course, feature development.
+
+The code review should be done before the code is pushed to staging or production.
+
+## Code Review Steps
+- Coder write new code on a new feature branch based off master
+- Coder push feature branch to Gitlab.com and open a merge request
+- Reviewer review/refactor code with coder
+- Reviewer comment 👍  and merge the request
+- Coder refactor code
+- Coder can ask for second review or push to staging
+
+## Review Standard
+The minimum standard we based on to review the code can be found [here](../../README.md#coding-standard).
```

Comment:
Just a suggestion, maybe you can shift the review standard to this page, as I notice that the review standard at that portion seem to be stand alone in that portion, I find that there a better connection if we can link it here.